7-React/src/components/TodoItem/TodoItem.driver.tsx

```diff
@@ -0,0 +1,101 @@
+import TodoItem from "./TodoItem";
+import { fireEvent, render, RenderResult } from "@testing-library/react";
+import dataHooks from "./data-hooks";
+import React from "react";
+
+class TodoItemDriver {
+  private wrapper?: RenderResult;
+  private todoDescription: string = "";
+  private deleteTaskMock: jest.Mock = jest.fn();
+  private editTextMock: jest.Mock = jest.fn();
+  private toggleDone: jest.Mock = jest.fn();
+
+  private editBtn = () => {
+    const element = this.wrapper?.queryByTestId(dataHooks.editBtn);
```

Comment:
check `this.wrapper` was initialized and throw error with proper message otherwise?